Setup/UpgradeData.php

```diff
@@ -50,4 +111,66 @@ protected function version_2_1_7(
             ]
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function version_2_3_2(
+        ModuleDataSetupInterface $setup
+    ) {
+        $eavSetup = $this->eavSetupFactory->create(['setup' => $setup]);
+        $eavSetup->removeAttribute(
+            \Magento\Customer\Model\Customer::ENTITY,
```

Comment:
we already have the \Magento\Customer\Model\Customer in usages there is no need in full class path here

Reply:
Thanks - done